docs/book/src/plugins/extending/external-plugins.md

```diff
@@ -120,26 +121,15 @@ Otherwise, Kubebuilder would search for the plugins in a default path based on y
 You can now use it by calling the CLI commands:
 
 ```sh
-# Initialize a new project with the external plugin named `sampleplugin`
-kubebuilder init --plugins sampleplugin/v1
+# Update the project configuration with the sample external plugin
+# The sampleexternalplugin adds Prometheus ServiceMonitor configuration
```

Comment:
I liked the idea, but it is added in all scaffolds by default.
Could we try to add a file that does not exist by default?
Maybe a Prometheus Config